test/logger.test.js

```diff
@@ -1369,7 +1370,7 @@ test('should not log the error if error handler is defined', t => {
         stream.once('data', line => {
           t.equal(line.level, 30, 'level is correct')
           t.equal(line.msg, 'request completed', 'message is set')
-          t.same(line.res, { statusCode: 500 }, 'status code is set')
+          t.same(line.res, { statusCode: 200 }, 'status code is set')
```

Comment:
Is no more set the status code to 500 by default?

Is the return code equals to 200 for this snippet?
```
fastify.setErrorHandler(function (error, request, reply) {
  this.log.error(error)
  reply.send({ success: false })
})
```

Reply:
Yes, we changed it in https://github.com/fastify/fastify/pull/2609/files.